docs/configuration.md

```diff
@@ -97,6 +97,12 @@ storage:
     accountname: accountname
     accountkey: base64encodedaccountkey
     container: containername
+    rootdirectory: /az/object/name/prefix
+    credentials:
+      type: client_secret
+      clientId: client_id_string
+      tenantId: tenant_id_string
```

Comment:
```suggestion
      clientid: client_id_string
      tenantid: tenant_id_string
```

No other config field names use camelCase. And underscores cannot be used either.
https://github.com/distribution/distribution/blob/0c958010ace2e0c2a87f1bf9915b7c74157dfb62/configuration/configuration.go#L16-L17

Reply:
cool, updated